t/TestWasmX.pm

```diff
@@ -270,6 +277,10 @@ add_block_preprocessor(sub {
                 $timeout += 30;
                 $timeout = max($timeout, 30);
             }
+
+        } elsif ($osname =~ m/darwin/ && $archname =~ m/arm64/) {
+            $timeout += 30;
+            $timeout = max($timeout, 30);
```

Comment:
With many intermittent tests removed from the macos suite, is this increase still necessary for sure?

Reply:
Indeed, it's not needed anymore.